Web/starter_project_iconics/pages/blog/singleBlog.tsx

```diff
@@ -0,0 +1,80 @@
+
+// type Blog  = {
+//     title : string,
+
+
+// }
+
+// interface props {
+//     title : Blog.title,
+
+// };
+
+const singleBlog = ({}) => {
+  return (
+    <div className="bg-white">
+        <h2 className="text-center blog-title pt-4">
+            The essential guide to Competitive Programming
+        </h2>
+        <p className="text-center uppercase blog-sub-title">
+            Programming, tech | 6 min Read
+        </p>
+        <div className="flex h-auto w-5/6  justify-center items-center mx-auto p-8">
+            <img src="/blogImg.png" className="mx-auto" alt="blog image"/>
+        </div>
+
+        <div className="mx-auto p-4">
+            <img src="/small-blog-img.png" className="mx-auto p-4"/>
+            <div className="">
+                <p className="mx-auto text-center uppercase font-normal blog-sub-title">chaltu kebede | software engineer</p>
+                <a className="text-center " href="#"><p className="text-center uppercase font-semibol text-blue-600 link-color">@chaltu_kebede</p></a>
+            </div>
+        </div>
+        <div className=" mx-auto pt-8 text-left w-4/6">
```

Comment:
If you are the same person that is doing the blog adding page, please check out prebuilt editors we can use to write the content of a blog (they act like google docs with the formatting and stuff)
check out how they save the data and try to implement it that way (maybe all you need to do is just print out the string that is saved as it might be saved with it's own style)